### ISSUE TITLE
Watching multiple files / directories?

### ISSUE THREAD

**Issue by contentfree** [xorcery/coyote#11]
I tried doing `coyote -w input1:output1 input2:output2` expecting it to watch all the given sources, but it only seems to recognize the first pair. Is this difficult to add to coyote?

**Comment by caseyohara** [comment]
@contentfree You're right, coyote wasn't designed to watch multiple input:output pairs. It's really meant for transitive dependency management for a single input point, building a tree of your input's dependencies, and then compiling to a single output.

I took a quick look last night at what would be involved in watching multiple files. While it doesn't look to be too difficult, I recommend you try using the require syntax to pull in your multiple dependencies. Check this out for the documentation around that: http://imulus.github.io/coyote/

Let me know what would work better for you.

/C

**Comment by contentfree** [comment]
What I was hoping to do was have coyote watch my CSS _and_ JS with a single
coyote watcher. As it stands, I have two terminals running two separate
watch commands. Would be great to have coyote just be able to take "watch
all of these for me" and Just Do It.

**Comment by caseyohara** [comment]
@contentfree Gotcha. That totally makes sense. Like this?:

`coyote -w input.css:output.css input.js:output.js`

**Comment by contentfree** [comment]
Exactly!

On Jul 26, 2013, at 8:16 AM, caseyohara notifications@github.com wrote:

@contentfree https://github.com/contentfree Gotcha. That totally makes
sense. Like this?:

coyote -w input.css:output.css input.js:output.js

—
Reply to this email directly or view it on
GitHubhttps://github.com/imulus/coyote/issues/11#issuecomment-21627171
.
